proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -206,6 +206,674 @@ function bufferNodeIs( test )
+
+  test.case = 'check I16x';
+  var src = new I16x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check I32x';
+  var src = new I32x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check I64x';
+  var src = new I64x( _.bigIntsFrom( [ 2, 3 ] ) );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check empty F32x';
+  var src = new F32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check F32x';
+  var src = new F32x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  test.case = 'check empty F64x';
+  var src = new F32x( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check F64x';
+  var src = new F64x( [ 2, 3 ] );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferNode */
+
+  if( Config.interpreter === 'njs' )
+  {
+    test.case = 'check empty BufferNode';
+    var src = new BufferNode.alloc(5);
+    var got = _.bufferBytesIs(src);
+    var expected = false;
+    test.identical( got, expected );
+
+    test.case = 'check BufferNode';
+    var src = new BufferNode.alloc(5, 4);
+    var got = _.bufferBytesIs(src);
+    var expected = false;
+    test.identical( got, expected );
+
+    test.case = 'check BufferNode from string';
+    var src = new BufferNode.from('str');
+    var got = _.bufferBytesIs(src);
+    var expected = false;
+    test.identical( got, expected );
+  }
+
+  /* BufferRaw */
+
+  test.case = 'check empty BufferRaw';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRaw';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check BufferRawShared';
+  var src = new BufferRawShared( 5 );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /* BufferView */
+
+  test.case = 'check BufferView';
+  var src = new BufferView( new U8x( [ 1, 2, 3 ] ).buffer );
+  var got = _.bufferBytesIs( src );
+  var expected = false;
+  test.identical( got, expected );
+
+  /**/
+
+  if( !Config.debug )
+    return;
+
+  test.case = 'check without argument';
+  var got = _.bufferBytesIs();
+  var expected = false;
+  test.identical( got, expected );
+
+  test.case = 'check extra arguments';
+  var got = _.bufferBytesIs( new U8x( [ 1, 2, 3 ] ), new BufferRaw( 5 ) );
+  var expected = true;
+  test.identical( got, expected );
+}
+
+//
+
+function constructorIsBuffer( test )
```

Comment:
Good test routine, 
It check every of bufferTyped alias and has not duplicates.